src/Orleans.Core/Shims/RuntimeStatisticsGroup.cs

```diff
@@ -1,34 +1,22 @@
 ﻿using System;
 using Microsoft.Extensions.Logging;
+using Orleans.Statistics;
 
 namespace Orleans.Runtime
 {
-    internal class RuntimeStatisticsGroup : IDisposable
+    internal class DummyHostEnvironmentStatistics : IHostEnvironmentStatistics
     {
-        private readonly ILogger logger;
-        public long MemoryUsage => 0;
         public long TotalPhysicalMemory => int.MaxValue;
-        public long AvailableMemory => TotalPhysicalMemory;
+
         public float CpuUsage => 0;
 
-        public RuntimeStatisticsGroup(ILoggerFactory loggerFactory)
-        {
-            this.logger = loggerFactory.CreateLogger<RuntimeStatisticsGroup>();
-        }
+        public long AvailableMemory => TotalPhysicalMemory;
 
-        internal void Start()
+        public DummyHostEnvironmentStatistics(ILoggerFactory loggerFactory)
```

Comment:
you can just inject `ILogger<DummyHostEnvironmentStatistics>` instead of `ILoggerFactory`